tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
+
+[ -z "$IMAGE_KEY" ] && error "Could not find '$IMAGE_KEY', qcow image has not been created"
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
+S3_BUCKET_NAME="fedora-testing-farm-image-import"
+AWS_CLI="aws"
+
+# Configure aws region and output format
+$AWS_CLI configure set default.region us-east-2
```

Comment:
Please move the region name to a variable

Reply:
Done